tests/cases.py

```diff
@@ -1379,6 +1386,42 @@ def _help_test_cli(self, args):  # noqa: D102
         return super()._help_test_cli(args)
 
 
+class InductiveModelTestCase(ModelTestCase):
+    """Tests for inductive models."""
+
+    mode = TRAINING
+    num_relations: ClassVar[int] = 7
+    num_entities_transductive: ClassVar[int] = 13
+    num_entities_inductive: ClassVar[int] = 5
+    num_triples_training: ClassVar[int] = 33
+    num_triples_inference: ClassVar[int] = 31
+    num_triples_testing: ClassVar[int] = 37
+
+    def _pre_instantiation_hook(self, kwargs: MutableMapping[str, Any]) -> MutableMapping[str, Any]:  # noqa: D102
+        dataset = create_inductive_dataset(
+            num_relations=self.num_relations,
+            num_entities_transductive=self.num_entities_transductive,
+            num_entities_inductive=self.num_entities_inductive,
+            num_triples_training=self.num_triples_training,
+            num_triples_inference=self.num_triples_inference,
+            num_triples_testing=self.num_triples_testing,
+            create_inverse_triples=self.create_inverse_triples,
+        )
+        training_loop_kwargs = dict(self.training_loop_kwargs or dict())
+        training_loop_kwargs["mode"] = self.mode
+        InductiveModelTestCase.training_loop_kwargs = training_loop_kwargs
+        # dataset = InductiveFB15k237(create_inverse_triples=self.create_inverse_triples)
+        kwargs["triples_factory"] = dataset.transductive_training
+        kwargs["inference_factory"] = self.factory = dataset.inductive_inference
```

Comment:
ah there it is :) I'll change self.factory to the `transductive_training`

Reply:
Is this correct? In an inductive setting we would want to evaluate using the inductive inference, right?

Reply:
btw, more lcwa tests fail because of the the DataLoader: 
```
train_data_loader = DataLoader(
            dataset=training_instances,
            num_workers=num_workers,
            batch_size=batch_size,
            drop_last=drop_last,
            shuffle=shuffle,
            pin_memory=pin_memory,
        )
```

a mock inductive dataset contains 33 triples w/ inverses - 66 sLCWA instances and 53 LCWA. The default batch size is 400, and the loop has the default parameter `drop_last=True`, so the only one batch of 66/53 instances is dropped and there are 0 batches in the DataLoader :)

Reply:
Explicitly set train_batch_size to 8 in https://github.com/pykeen/pykeen/pull/722/commits/efffae4eaead6c6c57464557152ad2dcba2bef1e

Reply:
> The default batch size is 400 (could it be found w/ AMO?),
If `batch_size=None`, the batch-size will be maximized (not sure whether this only happens for GPU though).